### ISSUE TITLE
Documentation fails with `jinja2.exceptions.UndefinedError: 'nbformat.notebooknode.NotebookNode object' has no attribute 'tags'`

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#1218]
See for example https://github.com/poliastro/poliastro/pull/1217

```
2021-05-11T22:27:27.1127366Z Traceback (most recent call last):
2021-05-11T22:27:27.1128453Z   File "/home/vsts/work/1/s/.tox/docs/lib/python3.8/site-packages/nbsphinx.py", line 1022, in parse
2021-05-11T22:27:27.1131332Z     rststring, resources = exporter.from_notebook_node(nb, resources)
2021-05-11T22:27:27.1132645Z   File "/home/vsts/work/1/s/.tox/docs/lib/python3.8/site-packages/nbsphinx.py", line 839, in from_notebook_node
2021-05-11T22:27:27.1133377Z     rststr, resources = super(Exporter, self).from_notebook_node(
2021-05-11T22:27:27.1134260Z   File "/home/vsts/work/1/s/.tox/docs/lib/python3.8/site-packages/nbconvert/exporters/templateexporter.py", line 384, in from_notebook_node
2021-05-11T22:27:27.1135194Z     output = self.template.render(nb=nb_copy, resources=resources)
2021-05-11T22:27:27.1136008Z   File "/home/vsts/work/1/s/.tox/docs/lib/python3.8/site-packages/jinja2/environment.py", line 1289, in render
2021-05-11T22:27:27.1136565Z     self.environment.handle_exception()
2021-05-11T22:27:27.1137291Z   File "/home/vsts/work/1/s/.tox/docs/lib/python3.8/site-packages/jinja2/environment.py", line 924, in handle_exception
2021-05-11T22:27:27.1137864Z     raise rewrite_traceback_stack(source=source)
2021-05-11T22:27:27.1138501Z   File "<template>", line 124, in top-level template code
2021-05-11T22:27:27.1139622Z   File "/home/vsts/work/1/s/.tox/docs/share/jupyter/nbconvert/templates/rst/index.rst.j2", line 1, in top-level template code
2021-05-11T22:27:27.1140414Z     {%- extends 'display_priority.j2' -%}
2021-05-11T22:27:27.1141297Z   File "/home/vsts/work/1/s/.tox/docs/share/jupyter/nbconvert/templates/base/display_priority.j2", line 1, in top-level template code
2021-05-11T22:27:27.1142188Z     {%- extends 'base/null.j2' -%}
2021-05-11T22:27:27.1143101Z   File "/home/vsts/work/1/s/.tox/docs/share/jupyter/nbconvert/templates/base/null.j2", line 26, in top-level template code
2021-05-11T22:27:27.1143766Z     {%- block body -%}
2021-05-11T22:27:27.1144501Z   File "/home/vsts/work/1/s/.tox/docs/share/jupyter/nbconvert/templates/base/null.j2", line 29, in block 'body'
2021-05-11T22:27:27.1145149Z     {%- block body_loop -%}
2021-05-11T22:27:27.1145906Z   File "/home/vsts/work/1/s/.tox/docs/share/jupyter/nbconvert/templates/base/null.j2", line 31, in block 'body_loop'
2021-05-11T22:27:27.1146887Z     {%- block any_cell scoped -%}
2021-05-11T22:27:27.1147513Z   File "<template>", line 17, in block 'any_cell'
2021-05-11T22:27:27.1148272Z   File "/home/vsts/work/1/s/.tox/docs/share/jupyter/nbconvert/templates/base/null.j2", line 84, in block 'any_cell'
2021-05-11T22:27:27.1149307Z     {%- block markdowncell scoped-%} {%- endblock markdowncell -%}
2021-05-11T22:27:27.1150059Z   File "<template>", line 153, in block 'markdowncell'
2021-05-11T22:27:27.1150887Z jinja2.exceptions.UndefinedError: 'nbformat.notebooknode.NotebookNode object' has no attribute 'tags'
```

**Comment by jorgepiloto** [comment]
Looks like a [new jinja2 release](https://github.com/pallets/jinja/releases) was made a few hours ago, @astrojuanlu

**Comment by jorgepiloto** [comment]
Yeah, that's it. I updated to the `Jinja2==3.0.0` version was able to reproduce previous issue. Should we impose `Jinja2<3.0.0` within `pyproject.toml` developers requirements for the moment?

**Comment by astrojuanlu** [comment]
Closed in #1219

**Comment by astrojuanlu** [comment]
Okay, this was reported upstream :) https://github.com/spatialaudio/nbsphinx/issues/563
